model-optimizer/mo/ops/convolution.py

```diff
@@ -233,6 +233,8 @@ def infer(node: Node):
         # this attribute should store lambda node: ... (check tf convolution extractor)
         if node.has_valid('get_group'):
             node['group'] = node.get_group(node)
+            if node['group'] == 1 and input_shape[node.channel_dims] != kernel_shape[node.input_feature_channel]:
```

Comment:
@GlebKazantaev , should we implement `get_group` function for TF operations and add this added line to the it?

Reply:
@yekruglov , @GlebKazantaev , any feedback?

Reply:
@lazarevevgeny Maybe we should move it to `extensions/front/tf/conv_ext.py`
Something like `'get_group': lambda node: node.group if 'group' in node and node.group is not None else node.in_node(0).shape[node.channel_dims] / node.kernel_shape[node.input_feature_channel],`

Reply:
Let's do this